### PR TITLE
Alt Text Inclusions

### PR DESCRIPTION
**Pull request by dereklh7**
Added missing alt text.

### CODE REVIEW

File: iis/configuration/system.applicationHost/sites/site/ftpServer/logFile.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ To support FTP publishing for your Web server, you must install the FTP service.
   
     > [!NOTE]
     > To support ASP.Membership authentication or IIS Manager authentication for the FTP service, you will also need to select **FTP Extensibility**.   
-    [![](logFile/_static/image8.png)](logFile/_static/image7.png)
+    [![Screenshot of the Internet Information Services folder's contained sub folders.](logFile/_static/image8.png)](logFile/_static/image7.png)
```
Comment (MeeraDi) [inline iis/configuration/system.applicationHost/sites/site/ftpServer/logFile.md:98]: Maybe: Screenshot of Internet Information Services sub folders.

File: iis/web-hosting/planning-the-web-hosting-architecture/shared-hosting-configuration.md [hunk 2]
```diff
@@ -129,15 +129,15 @@ Remote Administration is a key feature of the set of [IIS Management and Adminis
 4. [Web Deploy](../../publish/deploying-application-packages/index.md) lets users package and deploy Web applications and databases onto hosted environments.  
 The following diagram depicts the Remote Administration configuration.
 
-[![](shared-hosting-configuration/_static/image5.jpg)](shared-hosting-configuration/_static/image4.jpg)
+[![Diagram showing the remote administration from a user using Microsoft Visual Studio, which affects the Front and Back End servers.](shared-hosting-configuration/_static/image5.jpg)](shared-hosting-configuration/_static/image4.jpg)
```
Comment (MeeraDi) [inline iis/web-hosting/planning-the-web-hosting-architecture/shared-hosting-configuration.md:132]: Maybe: Diagram showing the Microsoft Visual Studio remote administration page which affects the Front and Back End servers.

File: iis/web-hosting/planning-the-web-hosting-architecture/shared-hosting-configuration.md [hunk 3]
```diff
@@ -129,15 +129,15 @@ Remote Administration is a key feature of the set of [IIS Management and Adminis
...
 
 <a id="EnhArc"></a>
 
 ### Enhanced Architecture (Optional Components)
 
 In some cases, service providers want to achieve the maximum degree of manageability, monitoring, and automation. These requirements can be met by adding a set of optional components to the core architecture described earlier in this article. This environment configuration may be required by hosters managing large datacenter facilities across multiple geographical locations. The following diagram depicts an enhanced architecture for Shared Hosting environments.
 
-[![](shared-hosting-configuration/_static/image7.jpg)](shared-hosting-configuration/_static/image6.jpg)
+[![Diagram showing the relationship being the Router, Front End servers, and a larger quantity of Back End servers.](shared-hosting-configuration/_static/image7.jpg)](shared-hosting-configuration/_static/image6.jpg)
```
Comment (MeeraDi) [inline iis/web-hosting/planning-the-web-hosting-architecture/shared-hosting-configuration.md:140]: relationship between

**Comment by dereklh7** [comment]
@Rick-Anderson @mcleblanc Final review for added missing alt text.

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**